profiles/ayushlohmod.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/37297939/178766012-d773a89b-8627-4ca7-bdad-5def06995121.png",
+    "issueId": 1229,
+    "name": "Ayush Lohmod",
+    "username": "ayushlohmod"
+  }
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient